### DISCUSSION TITLE
背景颜色 Background color

### DISCUSSION THREAD

**Discussion by chenghanzh** [Quorafind/Obsidian-Thino#604] [Q&A 问答]
这个主视图背景的颜色能换吗.......

![image](https://github.com/Quorafind/Obsidian-Thino/assets/50680244/72ee3418-0772-4350-9958-905df7323c3b)

**Comment by juestchaos** [comment]
你自己使用的主题，可以自己去主题中修改
或者自己针对 thino 增加 snippets
